cirq/contrib/noise_models/noise_models.py

```diff
@@ -173,16 +176,138 @@ def noisy_moment(self, moment: 'cirq.Moment',
+                    ops.Moment(
+                        ops.BitFlipChannel(self.bitflip_prob_map[q])(q)
+                        for q in system_qubits))
+            moments.append(moment)
+            return moments
+        else:
+            moments.append(moment)
+            if self.depol_prob_map:
+                gated_qubits = [
+                    q for q in system_qubits
+                    if moment.operates_on_single_qubit(q)
+                ]
+                if gated_qubits:
+                    moments.append(
+                        ops.Moment(
+                            ops.DepolarizingChannel(self.depol_prob_map[q])(q)
+                            for q in gated_qubits))
+            return moments
+
+
+def simple_noise_from_calibration_metrics(calibration: engine.Calibration,
+                                          depolNoise: bool = False,
+                                          dampingNoise: bool = False,
+                                          readoutDecayNoise: bool = False,
+                                          readoutErrorNoise: bool = False
                                          ) -> devices.NoiseModel:
-    """Creates a reasonable PerQubitDepolarizingNoiseModel using the provided
-    calibration data. This object can be retrived from the engine by calling
-    'get_latest_calibration()' or 'get_calibration()' using the ID of the
-    target processor.
+    """Creates a reasonable PerQubitDepolarizingWithDampedReadoutNoiseModel
+    using the provided calibration data.
+
+    Args:
+        calibration: a Calibration object (cirq/google/engine/calibration.py).
+            This object can be retrived from the engine by calling
+            'get_latest_calibration()' or 'get_calibration()' using the ID of
+            the target processor.
+        depolNoise: Enables per-gate depolarization if True.
+        dampingNoise: Enables per-gate amplitude damping if True.
+            Currently unimplemented.
+        readoutDecayNoise: Enables pre-readout amplitude damping if True.
+        readoutErrorNoise: Enables pre-readout bitflip errors if True.
+
+    Returns:
+        A PerQubitDepolarizingWithDampedReadoutNoiseModel with error
+            probabilities generated from the provided calibration data.
     """
+    if not any([depolNoise, dampingNoise, readoutDecayNoise, readoutErrorNoise
+               ]):
+        raise ValueError('At least one error type must be specified.')
     assert calibration is not None
-    rb_data: Dict['cirq.Qid', float] = {
-        qubit[0]: depol_prob[0] for qubit, depol_prob in
-        calibration['single_qubit_rb_total_error'].items()
-    }
-    return PerQubitDepolarizingNoiseModel(rb_data)
+    depol_prob_map: Dict['cirq.Qid', float] = {}
+    readout_decay_map: Dict['cirq.Qid', float] = {}
+    readout_error_map: Dict['cirq.Qid', float] = {}
+
+    if depolNoise:
+        depol_prob_map = {
+            qubit[0]: depol_prob[0] for qubit, depol_prob in
```

Comment:
I'm noticing now that this basically treats every non-measurement, non-virtual gate as equivalent to an RB gate.

@dstrain115, do you have a notion of how terrible this assumption is?